tests/testColorterms.py

```diff
@@ -0,0 +1,55 @@
+import os
+import numbers
+import unittest
+
+import lsst.utils.tests as utilsTests
+import lsst.pipe.tasks.photoCal as photoCal
+
+class ColortermOverrideTestCase(unittest.TestCase):
+    """Test that colorterms specific to CFHT override correctly"""
+    def setUp(self):
+        colortermsFile = os.path.join(os.environ["OBS_CFHT_DIR"],"config","colorterms.py")
+        self.photoCalConf = photoCal.PhotoCalConfig()
+        self.photoCalConf.colorterms.load(colortermsFile)
+
+    def testColorterms(self):
+        """Test that the colorterm libraries are formatted correctly"""
+        refBands = ["u", "g", "r", "i", "z"]
+        cfhtBands = ["u","g","r","i","z"]
+        for band in cfhtBands:
+            ct = self.photoCalConf.colorterms.getColorterm(band, photoCatName="e2v") # exact match
```

Comment:
Technically at least one more space is needed before the # according to the style guide.